### PR TITLE
Update  GPS Rescue Altitude, Heading and Tracking, and Altitude, Baro and RTH debugs

### PR DESCRIPTION
**Pull request by ctzsnooze**
Updates or adds the following debug headers and graph configs tp display the relevant fields properly in 4.4

Closed #592 and #572 which were never merged but did the same things.

- GPS Rescue Altitude
- GPS Rescue Heading
- GPS Rescue Tracking
- GPS Rescue Throttle PID
- RTH
- Altitude
- Barometer

Also dds Heading and Magnetometer options to `Add Graph` in the `Graph Setup` dialog so that people checking out their GPS or Mag can more readily load these options.

Would be grateful if this could be merged asap so that people can readily view the GPS Rescue debugs

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:1183]:
```suggestion
                            return (value / 100).toFixed(1) + 'm/s';
                        default:
                            return value.toFixed(0);
```

We need an unconditional check in java script.

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:1193]: Add a default case here also.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Sonar is really getting picky about changes not touched.

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js:1202]: thanks, done

**Pull request merged**
